clients/go/pkg/worker/jobStreamer.go

```diff
@@ -0,0 +1,116 @@
+// Copyright © 2018 Camunda Services GmbH (info@camunda.com)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package worker
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/camunda/zeebe/clients/go/v8/pkg/commands"
+)
+
+type jobStreamer struct {
```

Comment:
It would make sense to refactor and have a single job worker stream where the poller and the streamer both dump into. I don't know if I feel comfortable adding this dependency so late though :sweat_smile: But it looks nice :+1: